app/models/job.rb

```diff
@@ -1,2 +1,5 @@
 class Job < ActiveRecord::Base
+  belongs_to :company
+
+  validates :position, :posting_date, :description, :company_id, presence: true
```

Comment:
I actually like your idea of having the time validation with `today` as the default if none is given.